src/Kernel32.Shared/Kernel32.Helpers.cs

```diff
@@ -47,5 +48,60 @@ public static Win32ErrorCode GetLastError()
         {
             return (Win32ErrorCode)Marshal.GetLastWin32Error();
         }
+
```

Comment:
I agree maintaining it is painful if the docs were to change a lot. If we want folks to use the helper methods, and there is no intellisense docs for them, that seems unfortunate. My habit is to copy and paste the xml docs from the pinvoke method to the helper method and then adjust them as needed. But as I'm just grateful for your contributions, feel free to merge without them if you prefer.